galaxy_api/api/ui/viewsets/collection.py

```diff
@@ -19,13 +23,36 @@ def list(self, request, *args, **kwargs):
         self.paginator.init_from_request(request)
 
         params = self.request.query_params.dict()
+
+        param_tuples = self.request.query_params.lists()
+
+        log.debug('query params: %s', params)
+        log.debug('query param .lists(): %s', param_tuples)
+
         params.update({
             'offset': self.paginator.offset,
             'limit': self.paginator.limit,
         })
 
+        # [('q', ['ordered_by']), ('namespace', ['bar', 'sdfsd'])]
+        param_kwargs = {}
+        for param_tuple in param_tuples:
```

Comment:
This logic becomes overcomplicated. There are two different params parsing styles.
Please implement and use serializer instead.